kamailio/kamailio_dsiprouter.cfg

```diff
@@ -2256,30 +2244,7 @@ route[SERVERNATDETECT] {
 	return;
 }
 
-route[IS_RTPE_REQUIRED] {
-	#if ($si == "INTERNAL_IP_ADDR")
-	#	return;
-
-    # Sets the flag to show if RTPEngine is required
-    if (isbflagset(FLB_WS_DEVICE) || isbflagset(FLB_SRC_MSTEAMS) || isbflagset(FLB_SRC_MSTEAMS_ONHOLD) || isbflagset(FLB_DST_MSTEAMS)) {
-        # WebRTC destination or MSTEAMS source
-        setflag(FLT_USE_RTPE);
-    }
-    else if (allow_source_address(FLT_MSTEAMS)) {
-		setbflag(FLB_SRC_MSTEAMS);
-		setflag(FLT_USE_RTPE);
-   	}
```

Comment:
this should be set in NEXTHOP already but double check this during integration testing